generators/entity-client/templates/react/src/main/webapp/app/entities/entity-update.tsx.ejs

```diff
@@ -230,7 +230,7 @@ export const <%= entityReactName %>Update = (props: I<%= entityReactName %>Updat
           { !isNew ?
             <AvGroup>
               <Label for="<%= entityFileName %>-id"><Translate contentKey="global.field.id">ID</Translate></Label>
-              <AvInput id="<%= entityFileName %>-id" type="text" className="form-control" name="id" required readOnly/>
+              <AvInput id="<%= entityFileName %>-id" type="text" className="form-control" name="<%= primaryKey.name %>" required readOnly/>
```

Comment:
At angular I've removed the special case for id.
Removed the id filter, and added readOnly attribute when `field.id` is true.

Reply:
Good idea, but I think, that wont be enough, because we need to edit the id field - if it's a user assigned id, instead of the Long id, which generated by the database.